src/script/mls/MLSConversations.ts

```
@@ -50,25 +50,24 @@ export async function initMLSConversations(
 
   const mlsConversations = conversations.filter(isMLSCapableConversation);
 
-  await Promise.allSettled(
-    mlsConversations.map(async mlsConversation => {
-      const {groupId, qualifiedId} = mlsConversation;
+  for (const mlsConversation of mlsConversations) {
```

I think that behavior-wise, this changes how the joining works. 

From what I can understand:

- previously we would try to join all conversations (some could fail, it wouldn't matter)

but now:

- we try one by one and stop at the first failing conversation. 

Is that correct? 
Is that by design?

---

That's very valid point, i've added error handling (not to change the behaviour we just ignore error and log it into the console) https://github.com/wireapp/wire-webapp/pull/16293/commits/9b9027d0b0744f14fb7cd2a8bd6c7bf16de9832d

---

nice `onError` callback ❤️